### DISCUSSION TITLE
Place for TestFlight crash reports to be uploaded?

### DISCUSSION THREAD

**Discussion by OliNix** [JohanDegraeve/xdripswift#414] [General]
Is there a place where all TestFlight "Releasers" can share the crash reports which are submitted to them?

I imagine 99% of them will be unused, but they may be helpful when trying to track down a bug.

**Comment by OliNix** [comment]
The most prolific crashes from my TestFlight community for the last year for example:
<img width="1215" alt="Screenshot 2023-01-14 at 05 02 06" src="https://user-images.githubusercontent.com/16334811/212456791-3ec52b9d-d4d2-4b86-98d9-9b7f40e5ffa8.png">

**Comment by paulplant** [comment]
This UIKit crash seems to be pretty much fixed since iOS18 came out... 

I'll keep this discussion open because your screenshot could be useful as an example if releasers find common crashes happening a lot.
